src/applications/simple-forms/21-10210/pages/statementD.js

```diff
@@ -0,0 +1,22 @@
+// for Flow 1: self claim, vet claimant
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    'ui:title': 'Provide your supporting statement',
+    statement: {
+      'ui:title':
+        'Tell us what you think we need to know about the facts or circumstances relevant to your claim. Include any information that we don’t already have and that you think may support your claim.',
```

Comment:
I saw for `vetPersInfo` and `claimantPersInfo` you made custom components to determine the title. Can you not do the same here for these 4 statements? From what I can see the title is the only part that changes.
Just curious if we can simplify at all.

Reply:
The component trick worked for _root_-level 'ui:title', but didn't for any _field_-level 'ui:title'...

I'm gonna dig deeper into that to see if I can refactor the widget/component itself.

Reply:
Done!  We're back to one single statement page-file.  Now root-level ui:title is a component, and field-level ui:title is an updateSchema().

Thx for calling this out!  I must've forgotten to delete the field-level _static_ ui:title prop the first time I tried updateSchema a couple days ago.